### ISSUE TITLE
Groups Gone

### ISSUE THREAD

**Issue by darkphoenixff4** [loot/loot#1738]
I don't know what the heck happened, but my Groups Editor now only shows one group: Default.  Every other group is gone.  Somehow I doubt the expected order is complete random now.  This occurred after I upgraded to the latest 0.18.2.

**Comment by sibir-ine** [comment]
Please try updating your masterlist.

**Comment by darkphoenixff4** [comment]
Yeah, sorry, I realized shortly after posting that I completely forgot to
update my masterlist before attempting to edit the groups.  Everything went
back to normal once I did.
